OpenRA.Game/Graphics/SpriteLoader.cs

```diff
@@ -66,7 +65,7 @@ public static Sprite[] GetSprites(string filename, ISpriteLoader[] loaders, Shee
 
 		public static ISpriteFrame[] GetFrames(string filename, ISpriteLoader[] loaders)
 		{
-			using (var stream = GlobalFileSystem.Open(filename))
+			using (var stream = Game.ModData.ModFiles.Open(filename))
```

Comment:
Fixing this one is a further out of scope than my other requests, so i'm happy to leave this to the next PR.